packages/build/src/plugins/mutations.js

```diff
@@ -2,18 +2,18 @@
 
 const { set } = require('dot-prop')
 
-const { addErrorInfo } = require('../../error/info')
-const { EVENTS } = require('../events')
+const { addErrorInfo } = require('../error/info')
 
 const { getPropName } = require('./config_prop_name')
-const { removeProxies } = require('./track')
+const { EVENTS } = require('./events')
 
 // Apply a series of mutations to `netlifyConfig`.
 // Also normalize it.
 const applyMutations = function (netlifyConfig, configMutations) {
   configMutations.forEach(({ keys, value, event }) => {
     applyMutation({ netlifyConfig, keys, value, event })
   })
+  return netlifyConfig
```

Comment:
At the moment, we directly mutate `netlifyConfig`, but we should shallow copy it instead, which is why this is returned here.